examples/block_causal_attention/block_causal_attention.py

```
@@ -0,0 +1,593 @@
+    return fwd
+
+
+@tilelang.jit(out_idx=[2], pass_configs=_PASS_CFG)
+def _build_bwd_preprocess(batch: int, heads: int, seq_len: int, dim: int, dtype: str = "bfloat16"):
+    accum_dtype = "float32"
+    shape = [batch, seq_len, heads, dim]
+    block = 64
+
+    @T.prim_func
+    def prep(
+        O: T.Tensor(shape, dtype),
```

Ruff (0.15.20)

[error] Ambiguous variable name: `O`

(E741)